get-to-know-sense/incidents.md

```diff
@@ -1,4 +1,26 @@
 ---
 description: >-
   Describes the Incidents in SysKit Sense, how are they calculated and what can user do about them.
----
\ No newline at end of file
+---
+
+# Incidents Overview 
+
+The incidents are central part of the SysKit Sense. SysKit Sense comes with an advanced incident engine which monitors SharePoint Online and the networks where the SysKit Sense Agents are installed. Incident represents issues that are causing users to experience degraded Office 365 and SharePoint performance. When SysKit Sense determines that the incident has started or finished, it will send an email notification.

```

Comment:
not quite sure about the advanced though. advanced for me would be machine learning somthing something :D.